playbooks/common/openshift-cluster/upgrades/post_control_plane.yml

```diff
@@ -30,16 +34,15 @@
     - all_routers.results.returncode != 0
 
   - name: Update router image to current version
+    oc_edit:
+      kind: dc
+      name: "{{ item['labels']['deploymentconfig'] }}"
+      namespace: "{{ item['namespace'] }}"
+      content:
+        spec.template.spec.containers[0].image: "{{ router_image }}"
```

Comment:
just to capture it here and make sure, this will correctly report on 'changed', correct?